WordPress/Classes/Extensions/MediaService+Local.swift

```diff
@@ -93,21 +93,21 @@ extension MediaService {
         }
     }
 
-    /// Clear the locala Media directory of any files that are no longer in use by any managed Media objects.
+    /// Clear the local Media directory of any files that are no longer in use by any managed Media objects.
     ///
     /// - Note: These files can show up because of the app being killed while a media object
     ///   was being created or when a CoreData migration fails and the database is recreated.
     ///
     class func clearUnusedFilesFromLocalMediaDirectory(onCompletion: (() -> ())?, onError: ((Error) -> Void)?) {
-        purgeLocalMediaFiles(exceptMedia: NSPredicate.init(format: "blog != NULL", argumentArray: nil),
+        purgeLocalMediaFiles(exceptMedia: NSPredicate(format: "blog != NULL", argumentArray: nil),
```

Comment:
Nitpick, but you can remove the `argumentArray` parameter entirely from these calls.